wp-content/themes/twentyseventeen-wp20/page-about.php

```diff
@@ -7,26 +7,23 @@
 
 				<header class="entry-header">
 					<h1 class="entry-title">
-						<?php the_title(); ?>
+						<?php esc_html_e( 'People all over the world are celebrating the WordPress 20th Anniversary on May 27, 2022. Join us!', 'wp20' ); ?>
 					</h1>
 				</header>
 
 				<div class="entry-content">
-					<h1>
-						<?php esc_html_e( 'People all over the world are celebrating the WordPress 15th Anniversary on May 27, 2018. Join us!', 'wp20' ); ?>
-					</h1>
-
-					<img class="wp20-confetti-divider" src="<?php echo esc_url( get_stylesheet_directory_uri() ); ?>/images/confetti-divider.svg" alt="" />
-
-					<?php echo do_shortcode( '[wp15_meetup_events]' ); ?>
-
-					<img class="wp20-confetti-divider" src="<?php echo esc_url( get_stylesheet_directory_uri() ); ?>/images/confetti-divider.svg" alt="" />
-
-					<p class="wp10-nostalgia">
-						<?php printf(
-							wp_kses_data( __( 'Check out <a href="%s">this post about the WordPress 10th anniversary</a>.', 'wp20' ) ),
-							'https://wordpress.org/news/2013/05/ten-good-years/'
-						); ?>
+					<div class="entry-content-section">
+						<?php echo do_shortcode( '[wp15_meetup_events]' ); ?>
```

Comment:
```suggestion
						<?php echo do_shortcode( '[wp20_meetup_events]' ); ?>
```

I renamed it in 8b5dbc0caa0d9a80e8c1af02223e4982f435a899